### DISCUSSION TITLE
这里是陈梓聪的个人空间，有什么想法要告诉我吗？欢迎给我留言！

### DISCUSSION THREAD

**Discussion by giscus** [Chinesee/Chinesee#3] [留言]

**Comment by Codennnn** [comment]
有什么想跟我说的吗？在这里留下你的想法吧！

**Comment by SYJun404** [comment]
喜欢你的设计, 很美丽, 嘻嘻。
Blog: syjun.vip

**Reply by Codennnn** [reply]
你的博客做得也相当出色！

**Comment by feercc** [comment]
这个设计也太好了，我要借鉴一下了

**Comment by anghunk** [comment]
设计的很精致，非常有创意

**Comment by bh-lay** [comment]
2023年，还在坚持个人博客的越来越少了，收藏了。

**Comment by leon-fong** [comment]
>【数码产品】中的描述，span 标签内包含 span 标签导致换行时下方文字 backgroud 遮住了上方问题。

整体设计超棒，有计划支持暗黑模式吗？

**Reply by Codennnn** [reply]
我个人偏好浅色模式，因为它更利于阅读。虽然深色模式也很酷，但是会让样式控制变得更复杂，对于我的个人网站而言，还是希望保持简单一点。

**Reply by Codennnn** [reply]
你描述的「文字 backgroud 遮住了上方」，可以展示截图给我看吗，我没发现这个问题🤔

**Comment by ToulthG** [comment]
网站帅，本人更shuai！

**Comment by yakiisama** [comment]
从v2ex追溯进来，很棒的网站！

**Reply by yakiisama** [reply]
这里padding有点小问题，强迫症🥺

![](https://zoer.us/file/43b1532f3e9cd825ba5e3.png)

**Reply by Codennnn** [reply]
文字换行导致的，不受 padding 控制了

**Comment by pipuwong** [comment]
偶然中在 Github 有人推荐你的网站，竟然是东莞人，进来一看，风格和内容，调性也不错，我也是东莞，创业 摄影工作室8 年，目前是一名平面设计师，希望交个朋友。欢迎过来。pipuwong.com

**Reply by Codennnn** [reply]
你的网站做得很棒耶，我收藏了 ( •̀ ω •́ )y
如果你还有其他社交媒体账号，我很有兴趣关注你

**Reply by pipuwong** [reply]
Twitter：@pipuwong

**Comment by wurencaideli** [comment]
这个站点 https://green-wall.vercel.app/
和这个 https://vue-color-avatar.vercel.app/
设计的太好看了，小而美，羡慕了

**Comment by luckfunc** [comment]
看了你的vue color avatar后我用react复刻（抄）了一个。谢谢大佬

**Comment by zty42** [comment]
https://dribbble.com/leoku

404啦

**Comment by CaptainOfPhB** [comment]
希望站长加一个 “我的朋友” 类别，可以展示友联。

**Reply by CaptainOfPhB** [reply]
我的网站 https://captainofphb.me。

> 友联页面可参考 https://kaiyi.cool/friends

**Reply by CaptainOfPhB** [reply]
https://captainofphb.me

**Reply by Codennnn** [reply]
感谢你对我的网站感兴趣，不过目前我希望保持网站内容的简洁和清晰，所以可能不考虑添加其他友链，希望你理解哈。

你的个人网站做得相当不错呢！

**Comment by ghost** [comment]
略微帅气

**Comment by shenweiyan** [comment]
从 v2ex 认识并用上了 V2EX Polish，看到作者的求职点了进来，真是一个让人眼前一亮的网站，希望作者早日找到适合自己的工作！

**Comment by SivonLi** [comment]
大佬 请问你的简历模版可以开源吗

**Reply by Codennnn** [reply]
这个简历是我个人网站的一部分，暂时不考虑开源呢，其实这样式写起来很简单的。

**Reply by SivonLi** [reply]
> 这个简历是我个人网站的一部分，暂时不考虑开源呢，其实这样式写起来很简单的。

好的谢谢大佬回复

**Comment by shkmzzh** [comment]
想问一下 ，view-transition 在我的recat项目中不生效请问你是如何实现的？

**Reply by Codennnn** [reply]
可以参考这个脚本的实现：https://theme-toggle.rdsx.dev/

**Reply by shkmzzh** [reply]
可以了，感谢

**Comment by wolffyFeng** [comment]
请问下Netlify 国内限速吗

**Comment by poezhang** [comment]
好有趣的人啊 喜欢

**Comment by ChrisFeldmeier** [comment]
Hey, GreenWall App is cool, but I can not refresh my Github User Bio Description :/

**Comment by PungwjTNL** [comment]
偶然在一个论坛看到你的开源项目Vue Color Avatar，hhh这个风格的网站真的刷新了我的审美，之前一直混在一些资源博客都审美疲劳了。
